dcpy/models/connectors/__init__.py

```diff
@@ -14,10 +14,22 @@ def push(self, arg: _O, /) -> Any:
     def pull(self, arg: _I, /) -> Any:
         """pull"""
 
+    def list_versions(self, key: str, sort_desc: bool) -> list[str]:
+        return []
+
+    def latest_version(self, key: str) -> str:
+        return ""
+
+    def version_exists(self, key: str, version: str) -> bool:
+        return False
+
 
 class ConnectorDispatcher(Generic[_O, _I]):
     _connectors: dict[str, _ConnectorProtocol[_O, _I]]
 
+    def __getitem__(self, item):
```

Comment:
in the current implementation, we wrap the ConnectorDispatcher's methods. Not sure that really makes sense, tbh... might make more sense to just make the connectors retrievable, like so